1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,6 +1,89 @@
-function setAlarm() {}
 
-// DO NOT EDIT BELOW HERE
+// setInterval(countDown,1000);
+// const duration = 1;
+// function countDown(){
+//   duration--;
+//   console.log(countDown);
+//   if (duration === 0) {
+//     clearInterval(countDown) 
+
+//   }
+// }
+
+// function playAlarm(){
+// audio.play();
+// }
+
+
+
+// function setAlarm() 
+
+
+// // {
+// //   let countDown = setInterval(() => {
+// //     duration--;
+// //     console.log(duration);
+// //     if (duration === 0) {
+// //       clearInterval(countDown);
+// //     }
+// //   }, 1000);
+// // }
+
+
+
+
+let flasher = "";
+let timer = "";
+let timeRemainingInSeconds = 0;
+
+function flashBackground() {
+  flasher = setInterval(flashIt, 1000);
+  function flashIt() {
+    var x = Math.floor(Math.random() * 256);
+    var y = Math.floor(Math.random() * 256);
+    var z = Math.floor(Math.random() * 256);
+    var bgColor = "rgb(" + x + "," + y + "," + z + ")";
```

Comment:
Would be better not to hard code numbers directly in especially if you change these numbers later on it will make it difficult. Also, would it be better to call x, y, z after the colours for example r, g, b?